common/scripted_effects/00_scripted_effects.txt

```diff
@@ -5230,6 +5230,11 @@ remove_symptoms_effect = {
 		limit = { trait = malaise}
```

Comment:
My 6th mutation, not n'raqi yet.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79054446-f0aaa100-7c55-11ea-9387-550479ad6e56.png)
![image](https://user-images.githubusercontent.com/46384992/79054450-f56f5500-7c55-11ea-8402-cff1a51be8c1.png)

</details>